Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,33 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name;
+            string surname;
+            int age;
+            int weight;
+            int height;
+            int BMI;
+            int compter = 0;
```

Comment:
Declare each variable next to where you use them- or declare AND set their value. Don't do it all in one place, because it will force to scroll up and down is is harder to understand, requires to remember.